src/lib/Behat/Page/ContentViewPage.php

```diff
@@ -150,6 +156,27 @@ public function addLocation(string $newLocationPath): void
         $this->universalDiscoveryWidget->confirm();
     }
 
+    public function addTranslation(string $language, string $base): void
+    {
+        $this->getHTMLPage()->find($this->getLocator('addTranslationButton'))->click();
+        $this->dialog->verifyIsLoaded();
+        $this->translationDialog->selectNewTranslation($language);
+        if ($base != 'none') {
+            $this->translationDialog->selectBaseTranslation($base);
+        }
+        $this->dialog->confirm();
+    }
+
+    public function choosePreview(string $language): void
+    {
+        $this->getHTMLPage()->find($this->getLocator('previewDropdown'))->click();
+        $this->getHTMLPage()
+            ->findAll($this->getLocator('previewLanguage'))
+            ->getByCriterion(new ElementTextCriterion($language))
+            ->click();
+        $this->verifyIsLoaded();
```

Comment:
Just a quick comment, because in recent reviews I've been saying that calling `verifyIsLoaded` from a Page/Component is not recommeneded - this is a nice exception to this rule, because this action triggers a full page reload and we need to make sure it has ended - and verifyIsLoaded is great for that.

Reply:
Good to be aware. 📝